### PR TITLE
Blog post: Collaborative Lesson Development Training pilot announcement

### PR DESCRIPTION
**Pull request by tobyhodges**

### CODE REVIEW

File: _posts/2022/04/2022-04-06-lesson-development-training-pilot.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+## About the Training
+
+Aimed at groups of collaborators who want to start creating a new lesson,
+Collaborative Lesson Development Training will focus on three main topics:
+
+1. Good practice in lesson design and development
+2. Creating lesson websites with The Carpentries Workbench
+3. Collaboration skills
+
+Spread over six half-days, with an extended break between sessions 4 & 5,
```
Comment (kmomar) [inline _posts/2022/04/2022-04-06-lesson-development-training-pilot.md:27]: Could we add the schedule here if you have it ready? Perhaps we can make a visual of it?

**Review by kmomar — APPROVED** [review]

File: _posts/2022/04/2022-04-06-lesson-development-training-pilot.md (hunk 1 above)
Comment (tobyhodges) [reply _posts/2022/04/2022-04-06-lesson-development-training-pilot.md:27]: If I had time before I head out on parental leave, I would create a visual, but as it is I can really only offer the day-by-day schedule.

File: _posts/2022/04/2022-04-06-lesson-development-training-pilot.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+using [The Carpentries Workbench][workbench-blogpost], and 
+how to collaborate and build community around a lesson project. 
+
+Participants will apply these principles 
+as they are introduced during the training, 
+creating a sound foundation for their lesson 
+in a series of hands-on exercises, 
+trialling newly-developed content, 
+and preparing their lesson repository for collaboration.
+
```
Comment (tobyhodges) [inline _posts/2022/04/2022-04-06-lesson-development-training-pilot.md]:
```suggestion
### Planned Schedule
#### Day 1
- Introduction
- About your lessons
- Identifying your target audience
- Defining lesson objectives

#### Day 2
- The Carpentries Workbench
- Designing assessments 1

#### Day 3
- Designing assessments 2
- Example data and narrative

#### Day 4
- How to write a lesson
- How we operate
- Preparing to teach

**_break for participants to test their lesson content_**

#### Day 5
- Reflecting on trial runs
- Collaborating with newcomers	

#### Day 6
- Collaborating with people you already know
- Wrap-up
```

Comment (tobyhodges) [reply _posts/2022/04/2022-04-06-lesson-development-training-pilot.md]: @kmomar please commit the suggestion above if you think including this planned schedule would improve the post.

File: _posts/2022/04/2022-04-06-lesson-development-training-pilot.md (hunk 1 above)
Comment (kmomar) [reply _posts/2022/04/2022-04-06-lesson-development-training-pilot.md:27]: Understandable! I do think adding that when referencing specific sessions is a worthwhile addition. Thank you for making it :)

**Pull request merged**